h2o-core/src/main/java/water/api/RegisterV3Api.java

```diff
@@ -213,6 +213,14 @@ public void register(String relativeResourcePath) {
         "Return the model in the MOJO format. This format can then be interpreted by " +
         "gen_model.jar in order to perform prediction / scoring. Currently works for GBM and DRF algos only.");
 
+    RequestServer.registerEndpoint("makePDP",
+        "GET /3/Models/{model_id}/pdp", ModelsHandler.class, "makePDP",
```

Comment:
A rest api request for making a new PDP object should generally look like this:
`POST /3/PartialDependencePlots`